pr_check.sh

```diff
@@ -21,6 +21,7 @@ EXTRA_DEPLOY_ARGS="\
   --set-parameter ccx-notification-writer/IMAGE=quay.io/cloudservices/ccx-notification-writer \
   --set-parameter ccx-notification-db-cleaner/IMAGE=quay.io/cloudservices/ccx-notification-writer \
 "
+EXTRA_DEPLOY_ARGS="${EXTRA_DEPLOY_ARGS} --no-remove-resources notifications-engine"
```

Comment:
Pls add a comment about what this option exactly does. It bypasses the usual lower cpu/mem quota for deployment and dedicates more resources for deployment.

Reply:
PTAL: https://github.com/RedHatInsights/ccx-notification-writer/pull/503/commits/66e257d2ca92d5cd94da7401aea13d04200a9d1c